cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -35,6 +44,94 @@ cros_gralloc_driver::~cros_gralloc_driver()
 		drv_render_ = nullptr;
 		close(fd);
 	}
+
+	if (drv_ivshmem_) {
+		int fd = drv_get_fd(drv_ivshmem_);
+		drv_destroy(drv_ivshmem_);
+		drv_ivshmem_ = nullptr;
+		close(fd);
+	}
+	//pthread_join(tid, NULL);
+}
+
+void *cros_gralloc_driver::reload2(void *arg)
+{
+	int fd;
+	drmVersionPtr version;
+	char const *str = "%s/renderD%d";
+	const char *undesired[2] = { "vgem", nullptr };
+	uint32_t num_nodes = 63;
+	uint32_t min_node = 128;
+	uint32_t max_node = (min_node + num_nodes);
+	uint32_t j;
+	char *node;
+
+	const int render_num = 10;
+	const int name_length = 50;
+	int availabe_node = 0;
+	uint32_t gpu_grp_type = 0;
+	bool in_loop = true;
+
+	do {
```

Comment:
the workload seems too heavy. Can we firstly check if node counter change? Only the counter change, then continue check.